src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -110,13 +118,14 @@
     <!-- Explicitly pinned transitive dependencies with CVE vulnerabilities.-->
     <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets" Version="2.2.1" />
     <!-- The gRPC dependencies in this package are not compatible with older frameworks, like .NET Core 2.x -->
-    <PackageReference Include="Microsoft.DurableTask.Sidecar.Protobuf" Version="1.0.0" />
+    <PackageReference Include="Microsoft.DurableTask.Grpc" Version="1.1.0" />
+    <PackageReference Include="Grpc.Core" Version="2.46.6" /> <!-- Bring this in until we move to hosted RPC -->
   </ItemGroup>
 
   <!-- Common dependencies across all targets -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="2.0.0-preview.5" />
-    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.15.*" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.17.0" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="2.0.0-test.0" />
```

Comment:
DTFx.AzureStorage 2.0.0-test.0 is my test ADO feed built from the current azure-storage-v12 branch.

Reply:
Is there a reason why this is called `-test.0` instead of `preview.6` or something like that?

Reply:
Good question, yes it should be preview.6. I named it as test.0 in case we need to further update branch azure-storage-v12, eg merge from main again if we have new commits, new fixes merged to azure-storage-v12 like the pop receipt. And I don't want to occupy the version in ADO feed now. So I am using test.0 to pass CI and will update it back to preview.6 before merge.

Reply:
But if you agree that we can always increasing AS track2 version if there is a update, like preveiew.6, preview.7, I can update it now. ( actually I think it's a better idea